migrations/0061_wallet_custodian_check_custodian.up.sql

```diff
@@ -1,5 +1,5 @@
 ALTER TABLE wallet_custodian
 DROP CONSTRAINT IF EXISTS check_custodian,
 ADD CONSTRAINT check_custodian CHECK (
-    custodian IN ('brave', 'uphold', 'bitflyer', 'gemini', 'xyzabc')
+    custodian IN ('brave', 'uphold', 'bitflyer', 'gemini', 'zebpay')
```

Comment:
This must have been done as a new migration. Because the migration 61 had already been applied, these changes did not result in the changed state.

Now we are in a position where the database in inconsistent with migrations.

The production db has the migration 61, but the constraint, as expected, remains the one from the original migration:

```
Check constraints:
    "check_custodian" CHECK (custodian = ANY (ARRAY['brave'::text, 'uphold'::text, 'bitflyer'::text, 'gemini'::text, 'xyzabc'::text]))
```

Needs to be fixed manually.